portstyle.css

```diff
@@ -45,6 +46,7 @@ body {
 	background-image: url("https://images.unsplash.com/photo-1592750088063-96b411cd5895?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80");
 	background-repeat: no-repeat;
 	background-attachment: fixed;
+	height: 100vh;
 	width: 100%;
 	padding: 400px 0px;
```

Comment:
You're using height and padding to achieve the same thing, you can delete one as both is not necessary.

Reply:
Another situation of me forgetting to delete code just like before. I have removed the padding.